rpcd/playbooks/roles/rpc_maas/tasks/rabbitmq_user.yml

```diff
@@ -51,6 +46,6 @@
     write_priv: ".*"
     tags: "administrator"
     state: "present"
-  with_items: vhost_json
+  with_items: vhost_list.json
```

Comment:
vhost_list is a dictionary, one key of which is .json which points to an object constructed by interpreting the response as JSON. We did use from_json previously but it's not necessary anymore because the URI module can do it for us.